javascript/javascript1/week2/mainweek2.js

```diff
@@ -0,0 +1,109 @@
+/* Flight booking fullname function */
+
+function getFullname(firstname, surname, useFormalName) {
+    if (firstname === "") {
```

Comment:
Nice! Remember you also have the [logical OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators) if you wanted to do a check for an empty string for first and last name in the same line like: 
```js
if (firstname === "" || lastname === "") {
    return "please enter both first and last name "
}
```